### PR TITLE
add destination option for build_reloc.sh, build_deb.sh

### PR DESCRIPTION
**Pull request by syuu1228**
We need to change destination path to implement compatibility script.

### CODE REVIEW

**Comment by syuu1228** [comment]
@avikivity please review and merge this one prior to https://github.com/scylladb/scylla/pull/6896

File: reloc/build_deb.sh [hunk 1]
```diff
@@ -26,10 +32,10 @@ if [ ! -e $RELOC_PKG ]; then
...
 fi
 RELOC_PKG=$(readlink -f $RELOC_PKG)
-rm -rf build/debian
-mkdir -p build/debian/scylla-python3-package
-tar -C build/debian/scylla-python3-package -xpf $RELOC_PKG
-cd build/debian/scylla-python3-package
+BUILDDIR=$(readlink -f $BUILDDIR)
+mkdir -p $BUILDDIR/scylla-python3-package
+tar -C $BUILDDIR/scylla-python3-package -xpf $RELOC_PKG
+cd $BUILDDIR/scylla-python3-package
```
Comment (avikivity) [inline reloc/build_deb.sh:38]: Please quote shell variable references.

**Comment by syuu1228** [comment]
@avikivity fixed quote, please review again

**Comment by avikivity** [comment]
Oops, missed it, merged now.

**Pull request merged**